### DISCUSSION TITLE
Could we know the ranking of the proposal now?

### DISCUSSION THREAD

**Discussion by CoToYo** [camicroscope/GSOC#42] [Q&A] [answered]
Dear mentors,

The day of final project selections by GSOC is coming soon, so may I ask whether we can know the proposal rankings of each project now? It could help us a lot.
Thank you!!😄

**Comment by birm** [comment, marked as answer]
We are typically prevented from sharing any information before google does, including selection.

**Comment by CoToYo** [comment]
okay，thank you! 😁
